modules/rte_mis_school/rte_mis_school.module

```diff
@@ -30,7 +30,7 @@ function rte_mis_school_user_presave(UserInterface $user) {
   $roles = $user->getRoles();
   if (!empty($roles)) {
     // Now add the check for different roles.
-    if ((count($roles) === 1 & in_array('anonymous', $roles))
+    if ((count($roles) === 1 && array_intersect(['anonymous', 'authenticated'], $roles))
```

Comment:
@RohanSingh-inno Now we have added a new role, Called `school`. This role will be assigned to the Schools who are creating an account for the first time. Let's add a check of `school` role now.

Reply:
Ok